.github/workflows/ci-jest-coverage.yml

```
@@ -0,0 +1,15 @@
+name: 'coverage'
+on:
+    pull_request:
+        branches:
+            - master
+            - main
```

Should it do this instead, so it runs on any pull request that touches an app file? This is what we do on `ci-app.yml`

```suggestion
        paths:
          - app/**
          - .github/workflows/ci-jest-coverage.yml
```